added webinar to the docs (#580)
livechat/livechat-public-docs: content/getting-started/guides/index.mdx

  allowfullscreen
></iframe>

## Building apps webinar

> See our webinar recording on building LiveChat apps.

Reviewer: ```suggestion
To learn more about building apps for LiveChat, see our webinar where we go through the process step by step.
```

> Agenda:

Reviewer: We could link to those docs

> Agenda:

Reviewer: Are you sure you don't want to indicate what time in the video you start talking about something?

To learn more about building apps for LiveChat, see our webinar where we go through the process step by step.

Agenda:

> - creating an app in <a href="https://developers.livechatinc.com/console/" target="_blank">Developer Console</a> - 0:14:00.

Reviewer: ```suggestion
- creating an app in <a href="https://developers.livechatinc.com/console/" target="_blank">Developer Console</a> - 0:14:00
```

> - using [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/) - 0:18:55.

Reviewer: ```suggestion
- using [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/) - 0:18:55
```
